packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -55,17 +57,25 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     }
 
     public transformSchema(metadata: GraphbackCoreMetadata): GraphQLSchema {
-        const schema = metadata.getSchema()
+        let schema = metadata.getSchema();
         const models = metadata.getModelDefinitions();
         if (models.length === 0) {
             this.logWarning("Provided schema has no models. Returning original schema without any changes.")
 
             return schema;
-        }
+        };
+
+        schema = this.buildSchemaModelRelationships(schema, models);
+
+        // Print and rebuild to keep field and type descriptions
```

Comment:
Do we still need this workaround? If that related with your print schema?
I think we might put that info into issue that we have already as we need to be clear about what is happening.

Reply:
If I remove this line and, the generated does not keep the new (generated) descriptions, but keeps the ones from the input model.

This issue appears in both `printSchema` and `printSortedSchema`. I will create an issue.